backend/pkg/api/data_access/vdb_summary.go

```diff
@@ -871,6 +875,9 @@ func (d *DataAccessService) internal_getElClAPR(ctx context.Context, dashboardId
 	}
 	elIncomeFloat, _ := elIncome.Float64()
 	elAPR = ((elIncomeFloat / float64(aprDivisor)) / (float64(32e18) * float64(rewardsResultTable.ValidatorCount))) * 24.0 * 365.0 * 100.0
+	if math.IsNaN(elAPR) {
+		elAPR = 0
+	}
```

Comment:
could log an error here so we can debug, just to be safe

Reply:
I would leave it like it was written for the cl part for now and not add error logging.